aws/rhel-8.10-ga-x86_64/#config.json#

```diff
@@ -0,0 +1,6 @@
+o{
```

Comment:
Probably should not be part of the PR...

Reply:
I think I have my info/excludes set up badly :roll_eyes: